### ISSUE TITLE
electrum wallet drained

### ISSUE THREAD

**Issue by webinoak** [ecdsa/electrum-web#64]
I would like to recover my money.
I had opened my wallet and received bitcoin that I had purchased from Coinbase, then I proceeded to attempt to send part of that money as a payment. At that point I was instructed to download the latest version of the wallet.
Downloaded Electrum 4.0.0 from electrum.org.
The new wallet opened automatically and I proceeded with the payment. Before I clicked "send" I clicked back to my document to make sure everything was right and at that point Electrum sent the entire content of my wallet and I was left with a message of "not enough funds." The intended addressee received nothing. I have no idea where the money went.
![screenshot_2019-02-16 bitcoin explorerccc](https://user-images.githubusercontent.com/45165893/52895234-5bad3480-3185-11e9-9840-57c769c8d451.png)

**Comment by webinoak** [comment]
Does anyone have any idea what happened?

**Comment by webinoak** [comment]
I figured out that I was the victim of the very scam Electrum is warning about and the reason for the last upgrade. I clicked the link in the alleged warning and was taken to a replica of the download page, but admit I didn't look closely at the address, which was a dot com (as I learned after I looked at the properties of the phony Electrum 4.0.0 download today). Slick job. I hate to think how richly they were rewarded. It was a lot of money to me, but it was not so much compared to thousands of others, I'm sure. Lessons like that are way overpriced.
